src/target/riscv/riscv-013.c

```diff
@@ -279,16 +281,21 @@ dm013_info_t *get_dm(struct target *target)
 	return dm;
 }
 
-static uint32_t set_hartsel(uint32_t initial, uint32_t index)
+static uint32_t set_hartsel(uint32_t initial, int index)
```

Comment:
Can we call it the argument `hart_index`? For readability and also to avoid naming conflicts. `index()` is a function in one of the standard header files (`man 3 index`).

I'd also rename it to `set_dmcontrol_hartsel`.

```suggestion
static uint32_t set_dmcontrol_hartsel(uint32_t dmcontrol_val, int hart_index)
```

Reply:
Done.